File: 4th_sem/JAVA/assignment 3/q1.java
//Q.1) Write a program in Java to find the largest among 3 user entered nos. through command line.
class q1 {
    public static void main(String[] args) {
        if (args.length == 3) {
            int a = Integer.parseInt(args[0]);
            int b = Integer.parseInt(args[1]);
            int c = Integer.parseInt(args[2]);
            System.out.println("Largest number is : " + ((a > b) ? ((a > c) ? a : c) : ((b > c) ? b : c)));
        } else {
            System.err.println("Enter 3 numbers in parameter");
        }
    }
}

File: 4th_sem/SCHOOL/sem 2/assignments/assignment_7/q1.c
#include <stdio.h>

int add(int a, int b)
{ 
    return a+b;
}

void main()
{
    int a ,b;
    printf("enter 2 numbers to add: ");
    scanf("%d %d", &a, &b);
    int sum = add(a,b) ;
    printf("%d", sum);
}